dip-0004.md

```diff
@@ -132,6 +165,24 @@ To achieve this, It is suggested to first request a full masternode list from an
   6. Calculate the hash of “cbTx” and verify existence of this transaction in the block specified by “blockHash”. To do this, use the already received block header and the fields “totalTransactions”, “merkleHashes” and “merkleFlags” from the `MNLISTDIFF` message and perform a merkle verification the same way as done when a “MERKLEBLOCK” message is received. If the verification fails, abort the process and ask for diffs from another node.
   7. Store the resulting validated masternode list identified by “blockHash”
 
+## Tracking/Updating and verifying the active LLMQ sets based on MNLISTDIFF
+
+With the introduction of version 2 CbTx special transactions, a merkle root for the currently active LLMQ sets was added
+to the coinbase. This merkle root allows (SPV) clients to maintain the active LLMQ sets without the full chain.
+
+It is suggested to couple the house-/bookkeeping of active LLMQ sets with the masternode list bookkeeping described in
+the previous section and perform the following additional steps:
+
+  1. Create a copy of the active LLMQ sets which were given at "baseBlockHash". If “baseBlockHash” is all-zero, empty sets must be used.
+     As there are multiple LLMQ types.
```

Comment:
Looks like the "As there are multiple LLMQ types" should just be the end of the previous sentence.

Reply:
When rendered, it will actually be part of the previous sentence. I just don't like too long lines in Markdown :)

Reply:
Oh...now I see...that sentence shouldn't be there at all, updated last commit.